loggers/loggers_test.go

```diff
@@ -0,0 +1,89 @@
+package loggers
+
+import (
+	"encoding/json"
+	"math/rand"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+type FakeIoWriter struct {
+	// A list of strings that represent ints
+	ListOfInts []string
```

Comment:
Should you allocate the array to the expected size?

Reply:
The slice is easier from a testing perspective since I don't have to keep track of the latest entry (I can simply append).  Plus I dont think the performance is that bad with the current test.  We can always change it if we want to later.